drivers/dac/ltc268x/ltc268x.c

```diff
@@ -180,20 +180,21 @@ int32_t ltc2688_set_dither_toggle(struct ltc2688_dev *dev, uint16_t setting)
  * @param en - enable or disable dither mode
  * @return 0 in case of success, negative error code otherwise.
  */
-int32_t ltc2688_set_dither_mode(struct ltc2688_dev *dev, uint8_t channel,
+int32_t ltc268x_set_dither_mode(struct ltc268x_dev *dev, uint8_t channel,
 				bool en)
 {
 	uint16_t val = 0;
 	int32_t ret;
 
-	if (channel >= LTC2688_DAC_CHANNELS)
+	if (channel >= ltc268x_channels(dev->dev_id))
```

Comment:
seeing how ltc268x_channels is now an array, should be accessed using []: ltc268x_channels[dev->dev_id]
apply to the whole file

Reply:
this has been done